src/index.js

```diff
@@ -46,6 +46,11 @@ if (!ethers.utils.isAddress(deployedContractAddress)) {
   deployedContractAddress = '';
 }
 
+let tokenDecimals = urlSearchParams.get('decimals');
+if (!tokenDecimals) {
+  tokenDecimals = '18';
```

Comment:
we set this value as default, if no value for `decimals` is passed in the query param on the URL